src/ir/function-type-utils.h

```diff
@@ -23,9 +23,9 @@ namespace wasm {
 namespace FunctionTypeUtils {
 
 // Fills in function info from a function type
-inline void fillFunction(Function* func, FunctionType* type) {
-  func->params = type->params;
-  func->result = type->result;
+inline void fillFunction(Function& func, const FunctionType& type) {
```

Comment:
Why is the signature changed here? The downside is that we do tend to use pointers in most places, which means a bunch of extra `*`, which I think is less readable.

Reply:
I removed this commit.